src/gleam/pgo.gleam

```diff
@@ -0,0 +1,209 @@
+//// Postgresql client
+////
+//// Gleam wrapper around pgo library
+
+import gleam/atom.{Atom}
+import gleam/dynamic.{Dynamic}
+import gleam/string
+import gleam/io
+import gleam/map.{Map}
+import gleam/option.{Some}
+import gleam/uri.{Uri}
+
+/// Type representing the PID for a pool process
+/// NOTE this type should be switched to a general PID type when one is available
+pub external type Pid
```

Comment:
We could use the one in the OTP library?

Reply:
We could. It would mean including an external (and experimental) library so I wasn't sure about it.

This pid was the reason I was keen to introduce a Pid type into std lib, even if it didn't come with any process OTP functionality.

Having thought about it I would like to introduce the following into stdlib
```rust 
// gleam/process
pub external type Pid
```
That would be everything I would start with in that file.
A week or so back we discussed making typed pids a different thing, potentially a channel, that made sense to me.

Reply:
Sounds like bringing in the OTP library is the solution here. I still wish that using processes/pids didn't require to opt into OTP but maybe it's a level of complication that's not needed at the moment